packages/itwinui-css/src/checkbox/checkbox.scss

```diff
@@ -112,7 +112,7 @@
 
   &:disabled {
     --_iui-checkbox-svg-color: var(--iui-color-foreground-5);
-    --_iui-checkbox-border-color: var(--iui-color-background-border);
+    --_iui-checkbox-border-color: var(--iui-color-background-5);
```

Comment:
Changed so that the disabled border color isn't more prominent than the default state in high contrast mode.

**Old, using background border:**
![Screen Shot 2022-09-15 at 2 42 07 PM](https://user-images.githubusercontent.com/849817/190484244-59883103-42cd-48e0-804d-303bc3c447d3.png)

**New, using background 5:**
![Screen Shot 2022-09-15 at 2 43 31 PM](https://user-images.githubusercontent.com/849817/190484506-6d04392a-df60-4e5c-846c-1306cc744fbd.png)